tests_async/data/es256_privatekey.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
Make a commit to remove all these.

Reply:
I think these are sample data/credentials that were committed in previous versions too, should I still delete them?

Reply:
Yes, please remove them using git rm (no need to delete the data locally but they must be removed from this PR).

Reply:
Sorry-- I missed that this stuff is already checked in under tests/data and isn't data that you generated.

Could you make these tests use tests/data instead of tests_async/data and avoid having these files copied over?

Reply:
Resolved in the latest commit. Changed the directory paths to point to tests/data and got rid of the duplication in tests_async.